packages/fiori/src/themes/SideNavigationItemBase.css

```diff
@@ -29,7 +29,34 @@
 }
 
 :host([design="Action"]) {
-	color: var(--sapButton_TextColor);
+	color:  var(--sapButton_TextColor);
+}
+
+:host([design="Action"]) .ui5-sn-item:not(.ui5-sn-item-disabled):active [ui5-icon],
+:host([design="Action"]) .ui5-sn-item:not(.ui5-sn-item-disabled):active .ui5-sn-item-text {
+	color: var(--_ui5_side_navigation_active_text_color);
+}
+
+:host([design="Action"]) .ui5-sn-item {
+	border: var(--sapButton_BorderWidth) solid var(--sapButton_BorderColor);
+	box-sizing: content-box;
+}
+
+:host([design="Action"]:not([side-nav-collapsed])) .ui5-sn-item {
+	max-width: -moz-available;
+	max-width: -webkit-fill-available;
+	max-width: fill-available;
+}
+
+
+:host([design="Action"]) .ui5-sn-item:hover {
+	border: var(--sapButton_BorderWidth) solid var(--sapButton_Hover_BorderColor);
+	background: var(--sapButton_Hover_Background);
+}
+
+:host([design="Action"]) .ui5-sn-item:active {
+	border: var(--sapButton_BorderWidth) solid var(--sapButton_Active_BorderColor);
+	background: var(--sapButton_Active_Background);
```

Comment:
colors should be different on HCB and HCW

Reply:
You are wright, but the parameter is wrong. I reported it here:
https://www.figma.com/design/BOQ8ruQS2DtrHHoMXckJCX?node-id=4-1111#1137267763